### PR TITLE
Tests for Home page

### PR DESCRIPTION
**Pull request by KotliarovaK**

### CODE REVIEW

File: ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+	for (const [button, url] of buttonMap) {
+		await button.click();
+		const newPage = await driver.DriverContext.waitForEvent('page');
+		expect(newPage.url()).toContain(url);
+		await newPage.close();
+	}
+});
+
+test("Check 'See All News' button from the 'Company insights' block on the 'Home' page @Regression @QaAsAService @TSWEB-603", async () => {
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts:168]: ... from the 'Home' page @Regression @HomePage @TSWEB-1006

File: ui_tests_playwright/automation/identifiers/MainSite/pages/HomePage.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export default class HomePage {
+	static Info = 'Container-Info';
+	static IndustriesWeServe = 'Container-IndustriesWeServe';
+    static WhatWeDo = 'Container-WhatWeDo';
+	static PartnerTestimonials = 'Container-PartnerTestimonials';
+    static CaseStudies = 'Container-CaseStudies';
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/identifiers/MainSite/pages/HomePage.ts:6]: Problem with the format, or its only on remote?

File: ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+	const clutchButtons = await partnerTestimonialsContainer.getByTestId(MainSiteButtons.ClutchReviewArrow).all();
+
+	const buttonMap = new Map([
+		[clutchButtons[0], ClutchReviewLinks.MarkBeare],
+		[clutchButtons[1], ClutchReviewLinks.DerickDaily],
+        [clutchButtons[2], ClutchReviewLinks.DarrenCody],
+		[clutchButtons[3], ClutchReviewLinks.AnonymousMedicalDevice],
+        [clutchButtons[4], ClutchReviewLinks.AnonymousPeerToPeer],
+	]);
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts:158]: Code format

File: ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+	}
+});
+
+test("Check redirect by 'Clutch Review' button in 'Partner testimonials' container from the 'Home' page @Regression @HomePage @TSWEB-1006", async () => {
+	const partnerTestimonialsContainer = driver.getByTestId(HomePage.PartnerTestimonials);
+
+	const clutchReviewButton = await partnerTestimonialsContainer.getByTestId(MainSiteButtons.ClutchReviews);
+
+	await clutchReviewButton.click();
+
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts:149]: mb better also add a check that this tab is active?

**Review by Valerii-Kryvokhyzha — CHANGES_REQUESTED** [review]

File: ui_tests_playwright/automation/identifiers/MainSite/pages/HomePage.ts (hunk 2 above)
Comment (KotliarovaK) [reply ui_tests_playwright/automation/identifiers/MainSite/pages/HomePage.ts:6]: Yes, it's only on the remote. I'll try to fix it.

File: ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts (hunk 3 above)
Comment (KotliarovaK) [reply ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts:158]: I'll try to fix it.

File: ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts (hunk 4 above)
Comment (KotliarovaK) [reply ui_tests_playwright/automation/test/ui/MainSite/HomePage.test.ts:149]: Thank you! I'll add it

**Review by Valerii-Kryvokhyzha — APPROVED** [review]

**Review by DaniilVostrikov — APPROVED** [review]

**Comment by KotliarovaK** [comment]
moved all the changes to the other one because of a branch error.

**Pull request closed without merging**